pump/config.go

```diff
@@ -176,6 +177,15 @@ func (cfg *Config) Parse(arguments []string) error {
 	util.AdjustString(&cfg.DataDir, defaultDataDir)
 	util.AdjustInt(&cfg.HeartbeatInterval, defaultHeartbeatInterval)
 
+	if cfg.GC.Duration == 0 && cfg.GCStr == "" {
```

Comment:
this park looks confusing, when we both set int the config file and the command, the command line value should take effect instead of the one in the config file.

can we change
``` Duration as
type Duration string
``
and using the same object for both the config file and command line flag?

Reply:
Actually this part can make it to prioritize the config in the command. If we set type Duration to string, how do we get the actual duration? Unmarshall it every time?

Reply:
Can you explain it more clearly?

Reply:
> Actually this part can make it to prioritize the config in the command

yes, I got it wrong.

> If we set type Duration to string, how do we get the actual duration? Unmarshall it every time?

I think parsing it every time is ok here.


I am ok with this now, this will not block this pr.